### PR TITLE
[pt-br] Add concepts/configuration/manage-resources-containers.md

### PR DESCRIPTION
**Pull request by stormqueen1990**
## Changes
- Add translation for page concepts/configuration/manage-resources-containers.md.
- Add translation for two glossary entries:
  - eviction;
  - cgroups.

### CODE REVIEW

File: content/pt-br/docs/concepts/configuration/manage-resources-containers.md [hunk 1]
```diff
@@ -0,0 +1,872 @@
...
+comportando conforme o esperado, considere aumentar o limite (e possivelmente
+o requerimento) de memória para aquele contêiner.
+
+## {{% heading "whatsnext" %}}
+
+* Pratique [a criação de requerimentos de recursos de memória em contêineres e Pods](/docs/tasks/configure-pod-container/assign-memory-resource/).
+* Pratique [a criação de requerimentos de CPU em contêineres and Pods](/docs/tasks/configure-pod-container/assign-cpu-resource/).
+* Leia como a referência da API define um [contêiner](/docs/reference/kubernetes-api/workload-resources/pod-v1/#Container)
+  e seus [requerimentos de recursos](/docs/reference/kubernetes-api/workload-resources/pod-v1/#resources).
+* Leia sobre [quotas de projeto](https://xfs.org/docs/xfsdocs-xml-dev/XFS_User_Guide/tmp/en-US/html/xfs-quotas.html) no XFS.
```
Comment (stormqueen1990) [inline content/pt-br/docs/concepts/configuration/manage-resources-containers.md:870]: This link is currently broken. I found a manual page [here](https://man7.org/linux/man-pages/man8/xfs_quota.8.html). Can I replace the broken link for the manual page in this context?

File: content/pt-br/docs/reference/glossary/eviction.md [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+tags:
+- operation
+---
+
+Evicção é o processo de encerramento de um ou mais Pods em Nós.
+
+<!--more-->
+Existem dois tipos de evicção:
+* [Evicção por pressão no nó](/docs/concepts/scheduling-eviction/node-pressure-eviction/)
+* [Evicção iniciada pela API](/docs/concepts/scheduling-eviction/api-eviction/)
```
Comment (stormqueen1990) [inline content/pt-br/docs/reference/glossary/eviction.md:18]: These pages are not currently translated and I am unsure whether I should translate their descriptions in these links or just keep the page name in English until they get translated.

Comment (edsoncelio) [reply content/pt-br/docs/reference/glossary/eviction.md:18]: Hmm, I agree with translating the descriptions even the pages aren't translated, today we don't have a clear process to track the pages translations so at the least the descriptions will be in Portuguese.

File: content/pt-br/docs/concepts/configuration/manage-resources-containers.md (hunk 1 above)
Comment (edsoncelio) [reply content/pt-br/docs/concepts/configuration/manage-resources-containers.md:870]: Maybe you can open a PR/issue to fix the original docs and use the manual page here, I didn't see a problem with this approach :)

**Comment by edsoncelio** [comment]
@devlware PTAL :)

File: content/pt-br/docs/concepts/configuration/manage-resources-containers.md (hunk 1 above)
Comment (stormqueen1990) [reply content/pt-br/docs/concepts/configuration/manage-resources-containers.md:870]: Will open an issue to discuss the best way forward to fix the link!

Comment (stormqueen1990) [reply content/pt-br/docs/concepts/configuration/manage-resources-containers.md:870]: It looks like someone already fixed the link. I will copy the new link to the translation here.

**Pull request merged**